src/nav/full-stack-observability.yml

```diff
@@ -177,6 +177,1191 @@ pages:
             path: /docs/apm/new-relic-apm/getting-started/view-app-alert-information-apm
           - title: APM data security
             path: /docs/apm/new-relic-apm/getting-started/apm-agent-data-security
+      - title: Manage APM agents
+        pages:
+          - title: Agent data
+            pages:
+              - title: Real time streaming
+                path: /docs/agents/manage-apm-agents/agent-data/real-time-streaming
```

Comment:
each of these paths under the agents nav will need to be updated from `/docs/agents` to `/docs/apm/agents` and that should solve the missing left nav issue

Reply:
@LizBaker, so are you saying that these navigation links don't invoke the redirects the same way that redirects are invoked from within content files?

Reply:
@rhetoric101  the nav and redirects are kinda of separate things. when we attach nav menus to each node (aka, .mdx file in this context), we take the node's URL path (which is generated based on where it is in the file system) and we look for that URL path in each nav yaml file in `src/nav`. if we find a `path` value in a yaml file that matches the node's URL path, we attach that nav yaml menu to the node to display in the left nav. 

so since the agents nav items were brought into `full-stack-observability.yml` but the paths reflect all the nodes' old URLs / file paths, it's not going to attach a nav menu to the new nodes' paths.

so like Liz said, if you update all those `path` values to reflect the current URL / file paths for the agent `.mdx` files, that should attach the FSO nav to them all.

happy to hop on a call too if it's easier to walk through an example.

Reply:
@roadlittledawn, thanks for the clarification! I switched out all those old navigation prefixes with the new one and this worked!!